<?xml version="1.0"?>

<launch>
  <!-- arguments -->
  <arg name="path_input"      default="$(find offline_data)/file/exp01-02/slam_pose_graph.g2o"/> 
  <arg name="path_output"     default="$(find offline_data)/file/exp01-02/test_output.pcd"/> 

  <!-- launch offline_data -->
  <node name="combine_cloud" pkg="offline_data" type="combine_cloud" args="$(arg path_input) $(arg path_output)"/>
</launch>


<!-- 
<arg name="ds_size"         default="0.075"/> 
<arg name="range_threshold" default="10"/> 
<arg name="range_step"      default="1"/> 
-->

<!-- $(arg range_threshold) $(arg ds_size) $(arg range_step) -->